src/test/scala/io/iohk/ethereum/extvm/MessageHandlerSpec.scala

```diff
@@ -33,9 +37,16 @@ class MessageHandlerSpec extends AnyFlatSpec with Matchers with MockFactory with
       val (out, fut) = Source.queue[ByteString](1024, OverflowStrategy.dropTail).toMat(Sink.seq)(Keep.both).run()
       fut.pipeTo(probe.ref)
 
-      val gm = mock[GeneratedMessage]
-      (gm.toByteArray _).expects().returning(bytes.toArray[Byte])
```

Comment:
I believe we need to keep this assertion.

Reply:
This is not an assertion:
line 36 defines a mocked message
line 37 tells it what to return in a particular case

As soon as `toByteArray` cannot be mocked in a newer protobuf (it became `final`) 
I had to create an object itself returning the same array. And that's what I did below (line 43)